julia/v007_reuse_buffer.jl

```diff
@@ -0,0 +1,54 @@
+const NX::Int = 128
+const NY::Int = 128
+const MAX_ITERATIONS::Int = 1<<16
+
+const real = Float64
+
+const DX::real = 1.0/(NX-1)
```

Comment:
I know this is not the main point of the PR, but what is this `real` anotation doing here anyway? I'm getting
```julia
julia> DX::real = 1.0/(NX-1)
ERROR: TypeError: in set_binding_type!, expected Type, got a value of type typeof(real)
Stacktrace:
 [1] top-level scope
   @ REPL[1]:1
```

is `real` redefined somewhere to mean `Float64`?!

Reply:
In the line above the one you commented on, yes 😉

Reply:
🤦🏻

Reply:
oh, I see it didn't work in repl (because I used `Base.real` before and then I got all those errors) :D